.gitlab-ci.yml

```diff
@@ -4,7 +4,15 @@ stages:
   - deploy
   - status
 
+.run_only_if_file_changed_template:
+  only:
```

Comment:
Scheduled builds should also run unconditionally. That way, we can get updates during the weekly rebuilds.

Just wanted to add that in case someone decides to resurrect this pull request at some point in the future.